devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -44,6 +44,9 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: " RGBGU10Bulb50AU", deviceJoinName: "Aurora Light" //Aurora Smart RGBW
 		fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, 0300, FFFF, 1000", outClusters: "0019", manufacturer: "Aurora", model: "RGBBulb51AU", deviceJoinName: "Aurora Light" //Aurora RGBW GLS Lamp
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 1000, FFFF", outClusters: "0019", manufacturer: "Aurora", model: "RGBBulb51AU", deviceJoinName: "AOne Light" //AOne Smart RGBW GLS Lamp
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000, FC82", outClusters: "000A, 0019", manufacturer: "CWD", model: "ZB.M350rgbw-A001", deviceJoinName: "Smart RGBW Lamp" //Aurora Smart RGBW Lamp M350
```

Comment:
Please change Your fingerprints so they contain: manufacturer, model and a deviceJoinName only (profileId, inClusters,outClusters are not necessary).
According to new deviceJoinName writing rules deviceJoinName should look like: "Leedarson Light".
- add the raw description in a comment above the fingerprint (it can be found here: graphapi / devices -> select Your device -> Raw Description).
- add right device model name in a comment after the finger print.
- add a comment: // Leaderson - before a section of Your fingerprints